src/gluonts/model/predictor.py

```diff
@@ -21,6 +21,7 @@
 from pathlib import Path
 from pydoc import locate
 from tempfile import TemporaryDirectory
+import json
```

Comment:
I think normally we would want imports sorted alphabetically, but I don't mind too much.